test/integration/openid4ci_test.go

```diff
@@ -37,9 +40,11 @@ func TestOpenID4CIFullFlow(t *testing.T) {
 		issuerProfileID string
 		issuerDIDMethod string
 		walletDIDMethod string
+		isSD            bool
```

Comment:
Consider renaming this to `isSelectiveDisclosure` or at least add a comment explaining what the `SD` stands for - I don't think this is an obvious acronym for future readers.